.travis.yml

```diff
@@ -38,6 +38,7 @@ matrix:
             - libsdl2-ttf-dev
             - libnoise-dev
             - libopenal-dev
+            - vulkan-sdk
```

Comment:
I need to add a build option to disable this for now

Reply:
i'm confused... i've removed libsdl and all the other dependency libraries and re-enabled conan on master. why is are those libs still here?